bloom/generators/rpm/templates/ament_python/template.spec.em

```diff
@@ -15,7 +16,7 @@ Source0:        %{name}-%{version}.tar.gz
 @[if NoArch]@\nBuildArch:      noarch@\n@[end if]@
 
 @[for p in Depends]Requires:       @p@\n@[end for]@
-@[for p in sorted(BuildDepends + ['python%{python3_pkgversion}-devel'])]BuildRequires:  @p@\n@[end for]@
+@[for p in sorted(BuildDepends + ['python%{python3_pkgversion}-devel', 'python%{python3_pkgversion}-pytest'])]BuildRequires:  @p@\n@[end for]@
```

Comment:
Is pytest being added in our debian template?

Reply:
I don't think so. Maybe `dh_auto_test` is smart enough to skip the tests if `pytest` isn't installed? I could add some logic to do that if you like.

Reply:
What packages are motivating this inclusion? I just checked and neither ros_buildfarm nor bloom inject python3-pytest into the dependency chain unconditionally. I checked two python packages, demo_nodes_py and ros2pkg and they both depend explicitly on pytest via test_depend.

Reply:
There aren't any packages motivation the inclusion, the motivation is an error message during testing if it isn't installed because it's being called unconditionally.

I'll just add a conditional there to skip tests if we don't find pytest.

Reply:
I actually like this a lot better. If pytest isn't installed, you get:
```
/usr/bin/python3: No module named pytest
RPM TESTS SKIPPED
```

If it is installed, you get information about the pytest being used:
```
This is pytest version 4.6.9, imported from /usr/lib/python3.8/site-packages/pytest.py
setuptools registered plugins:
  pytest-mock-1.10.4 at /usr/lib/python3.8/site-packages/pytest_mock.py
  pytest-cov-2.8.1 at /usr/lib/python3.8/site-packages/pytest_cov/plugin.py
  colcon-core-0.5.9 at /usr/lib/python3.8/site-packages/colcon_core/pytest/hooks.py
  pytest-repeat-0.8.0 at /usr/lib/python3.8/site-packages/pytest_repeat.py
  pytest-rerunfailures-7.0 at /usr/lib/python3.8/site-packages/pytest_rerunfailures.py
```